src/v3/src/components/DuoWindow/DuoWindow.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+// import Duo from '../../../../../packages/vendor/duo_web_sdk';
+import Duo from '@okta/duo';
```

Comment:
for some reason lint is still giving an error that it cant find the module even after setting the alias in `preact.config`

Reply:
Take a look at `tsconfig.base.json` file, you may need to add this configuration there, there are examples in there you can mimic, I think that will resolve the issue you are seeing with the import.